tests/pages/pmmSettingsPage.js

```diff
@@ -191,7 +203,7 @@ module.exports = {
 
     customClearField(field) {
         I.appendField(field, '');
-        I.pressKey(['Shift', 'Home']);
+        I.pressKey(["Control", "a"]);
```

Comment:
why

Reply:
thats for clearing field. clearField method does not work correctly with React